src/pheromones/tests/BytesDTOUnitTests.cpp

```diff
@@ -4,7 +4,7 @@
 
 class BytesDTOFixture : public testing::Test {
   public:
-    const uint32_t m_byteReqId = 1;
+    const uint32_t m_byteReqId = 4;
```

Comment:
Was there a reason for this change?

Reply:
The reason the tests failed to notice the bug is that both values were the same, and the constructor interchanged these values. If we give a different value to each param, we can spot such issues.